src/widgets/index/index.ts

```diff
@@ -491,13 +595,15 @@ const index = (props: IndexProps): Index => {
         // be delayed. The render is triggered for the complete tree but some parts do
         // not have results yet.
 
-        if (widget.render && derivedHelper!.lastResults) {
+        if (widget.render) {
```

Comment:
This might be a breaking change, since it's possible  `render` is called without results, which might throw errors in the widgets themselves.

Reply:
There's no behavior change, it's taken care of here now:

https://github.com/algolia/instantsearch.js/blob/a8d42cdfe3e767887fdb52172fc1adadc6ce000d/src/widgets/index/index.ts#L558-L560